docs/howto/shared-cache.rst

```diff
@@ -0,0 +1,73 @@
+.. _howto-shared-cache:
+
+Cache intermediate build artefacts
+==================================
+
+Because Charmcraft builds Python packages from source rather than using pre-built
+wheels, the initial builds of charms can take a while. However, future builds of
+the same charm are sped up significantly by reusing the built wheels. When installed
+as a snap, Charmcraft automatically caches these wheels in the
+``~/snap/charmcraft/common/cache`` directory. However, in some cases, it may be
+beneficial to change this directory. This can be done by setting the
```

Comment:
With how-tos we want to prioritise tangible steps, so I think we should slice off "This can be done..." and place it in its own section.